src/store/model/ldlc.ts

```diff
@@ -0,0 +1,244 @@
+import {Store} from './store';
+
+export const Ldlc: Store = {
+  currency: '€',
+  labels: {
+    inStock: {
+      container: '.stock-1',
+      text: ['en', 'stock'],
+    },
+    maxPrice: {
+      container: '.price .price',
+      euroFormat: true,
+    },
+    outOfStock: {
+      container: '.stock-4 .stock-7',
+      text: ['RUPTURE'],
+    },
+  },
+  links: [
+    {
+      brand: 'evga',
+      model: 'xc3 ultra',
+      series: '3080',
+      url: 'https://www.ldlc.com/fr-be/fiche/PB00369303.html',
+    },
```

Comment:
```suggestion
    {
      brand: 'test:brand',
      model: 'test:model',
      series': 'test:series',
      url: 'https://www.ldlc.com/fr-be/fiche/PB00317660.html',
    },
    {
      brand: 'evga',
      model: 'xc3 ultra',
      series: '3080',
      url: 'https://www.ldlc.com/fr-be/fiche/PB00369303.html',
    },
```

We should add a test model in here.